<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OivObjectenDockWidgetBase</class>
 <widget class="QDockWidget" name="OivObjectenDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>OIV Objecten</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>360</width>
         <height>534</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>U heeft het volgende repressief object geselecteerd:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="formelenaam">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="object_id">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lengte_label">
            <property name="text">
             <string>Lengte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lengte">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="oppervlakte_label">
            <property name="text">
             <string>Oppervlakte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="oppervlakte">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="straal_label">
            <property name="text">
             <string>Straal hulpcirkel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="straal">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="objectgegevens">
            <property name="text">
             <string>Objectgegevens bewerken</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="object_tekenen">
            <property name="text">
             <string>Object terrein tekenen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="object_symbolen">
            <property name="text">
             <string>Object symbolen tekenen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_f">
            <property name="text">
             <string>Object verwijderen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="terugmelden">
            <property name="text">
             <string>Terugmelden BGT en/of BAG</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="terug">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Gereed</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
